pkg/store/instance_windows.go

```diff
@@ -115,6 +116,33 @@ func GetWslStatus(instName string) (string, error) {
 	return instState, nil
 }
 
-func GetSSHAddress(_ string) (string, error) {
-	return "127.0.0.1", nil
+// GetSSHAddress runs a hostname command to get the IP from inside of a wsl2 VM.
+//
+// Expected output (whitespace preserved, [] for optional):
+// PS > wsl -d <distroName> bash -c hostname -I | cut -d' ' -f1
+// 168.1.1.1 [10.0.0.1]
+// But busybox hostname does not implement --all-ip-addresses:
+// hostname: unrecognized option: I
+func GetSSHAddress(instName string) (string, error) {
+	distroName := "lima-" + instName
+	// Ubuntu
+	cmd := exec.Command("wsl.exe", "-d", distroName, "bash", "-c", `hostname -I | cut -d ' ' -f1`)
```

Comment:
```suggestion
	cmd := exec.CommandContext(ctx, "wsl.exe", "-d", distroName, "bash", "-c", `hostname -I | cut -d ' ' -f1`)
```
w/ `context.TODO()` for now